Explorer/Assets/DCL/SDKComponents/AvatarModifierArea/Components/AvatarModifierAreaComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Google.Protobuf.Collections;
+using System.Collections.Generic;
+
+namespace DCL.SDKComponents.AvatarModifierArea.Components
+{
+    public struct AvatarModifierAreaComponent
+    {
+        public readonly HashSet<string> ExcludedIds;
+
+        public AvatarModifierAreaComponent(RepeatedField<string> excludedIds)
```

Comment:
Here too RepeatedField

Reply:
Tackled in [this commit](https://github.com/decentraland/unity-explorer/pull/336/commits/2ff94b5bfe636a43c73c3a8616c9bca7b55d70cb)